app/tray/Account/Account/index.js

```diff
@@ -334,6 +356,9 @@ class _AccountBody extends React.Component {
       } else if (req.type === 'signTypedData') {
         accountViewTitle = 'Sign Data'
         // accountViewIcon = svg.sign(17)
+      } else if (req.type === 'signErc20Permit') {
+        accountViewTitle = 'Sign Token Spend Permit'
```

Comment:
I think we want to present this pathway of approving token spends and the old pathway (using an explicit 'approve') transaction the same way to users, rather than having two different requests type, both should look the same to users. we should discuss this part of the design though to figure out the best way forward here